scipy/stats/tests/test_stats.py

```diff
@@ -900,6 +900,22 @@ def test_index_error(self):
             0.0007835762419683435])
         assert_raises(ValueError, stats.spearmanr, x, y, axis=2)
 
+    def test_alternative(self):
+        x1 = [1, 2, 3, 4, 5]
+        x2 = [5, 6, 7, 8, 7]
+        expected = (0.82078268166812329, 0.088587005313543798)
```

Comment:
@DominicChm where did this come from?

Reply:
That was copied from a working test above. I'll add a comment to indicate so.